gen/build_deploy/bash/dcos-launch.spec

```diff
@@ -1,4 +1,12 @@
-a = Analysis(['test_util/launch.py'])
+# Warning:
+#   - hidden import must be used as there is a bug in pyinstaller
+#     https://github.com/pyinstaller/pyinstaller/issues/2185
+#   - data must be decalared explicitly if not a .py file
+#   - Building will suck up the local SSL .so and package it
+#     with the final exe. Ensure build system has OpenSSL 1.0.2g or greater
```

Comment:
TODO: Add a pre-req checker that verifies OpenSSL 1.0.2g+
It is highly unlikely that someone will read this, the first time they are excitedly trying dcos-launch.